core/regions/src/main/java/software/amazon/awssdk/regions/ServiceMetadata.java

```diff
@@ -78,25 +107,6 @@ static ServiceMetadata of(String serviceEndpointPrefix) {
         return metadata == null ? new DefaultServiceMetadata(serviceEndpointPrefix) : metadata;
     }
 
-    default String computeEndpoint(String endpointPrefix,
```

Comment:
This seems like a breaking change. Should we deprecate it instead?

Reply:
Nvm, saw it in the description. Can we mention it in the change log entry?

Reply:
I am almost certain that no one is using it. I'd much rather just delete it, because anyone using it is almost certainly doing it wrong.